### DISCUSSION TITLE
Info about not fully indexed repos may show same name twice

### DISCUSSION THREAD

**Discussion by Jackenmen** [community/community#10164] [Other Feature Feedback, Questions, & Ideas] [answered]
1. Go to https://cs.github.com
2. Search for `repo:linuxmint/cinnamon xkb` and end up at https://cs.github.com/linuxmint/cinnamon/blob/d66ff9c88b0882fdba6a3a4d6c0f5de36393c593/src/st/st-im-text.c?q=repo%3Alinuxmint%2Fcinnamon+xkb#L65
3. Click on "Search for `repo:linuxmint/cinnamon xkb` in this repo" and end up at https://cs.github.com/linuxmint/cinnamon?q=repo%3Alinuxmint%2Fcinnamon%20xkb
4. See warning message "Some selected repos (linuxmint/cinnamon, linuxmint/cinnamon) are not fully indexed yet. Results will be incomplete."

**Comment by mbellani** [comment, marked as answer]
Hi @jack1142 thanks for the report, this is now fixed
